core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerHttpClientBuilder.java

```diff
@@ -50,14 +50,26 @@ public CircuitBreakerHttpClientBuilder(
     @Override
     public CircuitBreakerHttpClient build(Client<HttpRequest, HttpResponse> delegate) {
         if (needsContentInStrategy) {
-            return new CircuitBreakerHttpClient(delegate, circuitBreakerMapping(), strategyWithContent());
+            return new CircuitBreakerHttpClient(delegate, mapping(), strategyWithContent());
         }
 
-        return new CircuitBreakerHttpClient(delegate, circuitBreakerMapping(), strategy());
+        return new CircuitBreakerHttpClient(delegate, mapping(), strategy());
     }
 
     @Override
     public Function<Client<HttpRequest, HttpResponse>, CircuitBreakerHttpClient> newDecorator() {
         return this::build;
     }
+
+    // Methods that were overridden to change the return type.
+
+    @Override
+    public CircuitBreakerHttpClientBuilder circuitBreakerMapping(CircuitBreakerMapping mapping) {
+        return (CircuitBreakerHttpClientBuilder) super.circuitBreakerMapping(mapping);
```

Comment:
ditto

Reply:
Just replaced with `mapping(mapping)` which is simpler.